server/src/main/resources/ehcache.xml

```diff
@@ -1,15 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <ehcache
-    maxBytesLocalHeap="300M"
+    maxEntriesLocalHeap="100000"
     >
 
+	<sizeOfPolicy maxDepth="100" maxDepthExceededBehavior="abort"/>
+
     <defaultCache 
+    	maxEntriesLocalHeap="100000"
         eternal="false" 
         timeToIdleSeconds="86400" 
```

Comment:
Is a 24 hour expire time proper? Is there a way to make this configurable from the candlepin.conf file?